frontend/src/components/Home.vue

```diff
@@ -42,7 +42,7 @@
           now invited to fill out and submit a New School Application. We have
           created a draft for you. Click
           <router-link
-            :to="{ path: '/school-application' }"
+            :to="{ path: '/school-application/APP-1048' }"
```

Comment:
100% - This was the "Alert message" as the top of the homepage that is all hardcoded, there is a ticket to made this alert data driven in the future, the link wasn't working since i added :id to the params, and of course didn't have any mock data connected to this component, so just needed to mock the id for now.